src/main/java/com/rmn/qa/servlet/AutomationTestRunServlet.java

```diff
@@ -327,13 +328,16 @@ public static void startNodes(VmManager ec2, String uuid,int threadCountRequeste
             log.info(String.format("%d instances started", instances.size()));
             // Reuse the start date since all the nodes were created within the same request
             Date startDate = new Date();
+            List<AutomationDynamicNode> createdNodes = Lists.newArrayList();
             for(Instance instance : instances) {
```

Comment:
If its empty, the for loop block just won't get executed.  Also, we have a check deeper in the stack to where if the size isn't greater than 0, an exception will be thrown.